src/roswire/proxy/roslaunch/config/node.py

```diff
@@ -20,9 +20,9 @@ class NodeConfig:
     namespace: str
     name: str
     typ: str
-    package: str
     executable_path: str
     executable_type: ExecutableType
+    package: Optional[str] = attr.ib(default=None)
```

Comment:
This will break existing client code, including parts of ROSDiscover. Are there cases where a node doesn't have an associated package?

Reply:
Fixed